src/analysis/types.rs

```diff
@@ -13,12 +13,12 @@ pub trait IsPtr {
 impl IsPtr for Field {
     fn is_ptr(&self) -> bool {
         if let Some(ref c_type) = self.c_type {
-            return c_type.contains('*')
+            return c_type.contains('*');
```

Comment:
No need for `return` in here.

Reply:
Thanks, I don't notice that many of unneeded early returns here